src/main/java/Duke.java

```diff
@@ -4,18 +4,19 @@
 
     public static void main(String[] args) {
         Scanner userInput = new Scanner(System.in);
-        String command;
         String HORIZONTAL_LINE = "____________________________________________________________";
```

Comment:
perhaps this could be in small caps? you could use camelCase convention:)

Reply:
I was unsure if this should be in all capital or camelCase convention, thank you for pointing this out to me! I will change it in my next commit :)